src/components/ManaCalculator/components/ValidatorSettings.tsx

```diff
@@ -52,7 +52,9 @@ export function ValidatorSettings() {
           0,
         )}
         onChange={(e) =>
-          handleAttractedNewDelegatedStakeChange(Number(e.target.value))
+          handleAttractedNewDelegatedStakeChange(
+            toMicro(Number(e.target.value)),
```

Comment:
Why to micro?

Reply:
fromMicro returns the value in Mana/IOTA, i dont see why we would want to re-convert it to micro, the units in the mana calculator are Mana/IOTA/Shimmer and not MicroMana/Micro/Glow

Reply:
Seems like the calculations are all done in micros, even the iota and smr supply are in "micros", all but this one

Reply:
https://github.com/iotaledger/iota-wiki/blob/main/src/components/ManaCalculator/components/ValidatorSettings.tsx#L25

Reply:
Validator Card [L26](https://github.com/iotaledger/iota-wiki/blob/main/src/components/ManaCalculator/components/ValidatorCard.tsx#L26) [L32](https://github.com/iotaledger/iota-wiki/blob/main/src/components/ManaCalculator/components/ValidatorCard.tsx#L32)